### DISCUSSION TITLE
Repl docs!

### DISCUSSION THREAD

**Discussion by kokonut27** [ReplAPI-it/ReplAPI.it-NodeJS#35] [General Community]
OoooOhHHhh!!! You finished da docs!!! OoOOOhHHH! xD Awesome docs btw! ;D

**Comment by rayhanadev** [comment]
Haha @JBYT27 yep I finished the docs

Not that it means anything I'm working on a new version of the package that will require me to practically start documentation from scratch xD.

If you want to check out what I have so far (it's in the final stages) take a look at [this](https://github.com/RayhanADev/ReplAPI.it/tree/RayhanADev-Major-Refactors) branch.

**Reply by kokonut27** [reply]
lmao, okie! thanks!

**Reply by rayhanadev** [reply]
Lol I just rolled out version 2.0.0 of ReplAPI.it 🎉

**Comment by rayhanadev** [comment]
Hey just saddling off of this conversation, if anyone wants to help out with documenting the version 2.x line I would be more than happy to get the help!
